ludwig/data/preprocessing.py

```diff
@@ -1312,6 +1314,10 @@ def _preprocess_file_for_training(
             random_seed=random_seed
         )
 
+        # save split values for use by visualization routines
+        split_fp = os.path.splitext(dataset)[0] + '_split.csv'
```

Comment:
I would create a function in data utils, that given a dataset path return the split file path.
It will make it easy for other parts of the code ot obtain the correct path without knowing the actual logic.
Also, to be similar to what we do for metadata where se use `.meta.json` here we can use `.split.csv` as extension.